Project-Timer-frontend/src/app/login-page/login-page.component.scss

```diff
@@ -0,0 +1,17 @@
+
+.example-form {
+    margin-top: 50px;
+    margin-left: 50px;
+    min-width: 150px;
+    max-width: 500px;
+    width: 100%;
+  }
+  
+  .example-full-width {
+    margin: 5px;
+    width: 100%;
+  }
```

Comment:
jak wyżej, nazwy do zmiany  example nic nie mówi :)  jeśli wysyłasz coś na gita to ma być już przez ciebie sprawdzone, zapoznaj się z BEM i daj znać czy Ci pasuje

Reply:
done